### PR TITLE
Implemented post display feature

### PR DESCRIPTION
**Pull request by otadewosjan**
JSON file is loaded and then its content is displayed as posts using list tags.

### CODE REVIEW

**Review by mjendza — COMMENTED** [review]
Please use JQuery

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Facebook</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+    </head>
+    <body onload="loadJsonDoc()">
```
Comment (mjendza) [inline index.html]: Is this the best place to run script ?

File: script.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+function loadJsonDoc() {
+    let myDoc;
```
Comment (mjendza) [inline script.js]: what version of JavaScript do you use in project ?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    let xhr = new XMLHttpRequest();
+    xhr.open("GET", "http://jsonplaceholder.typicode.com/posts", true);
+
+    xhr.onprogress = function () {
+        console.log('LOADING...', xhr.readyState);
+    };  
+
+    xhr.onload = function() {
+        if (this.status === 200) {
+            myDoc = JSON.parse(this.responseText);
```
Comment (mjendza) [inline script.js]: what means myDoc ? why this variable is outside this function scope ?

**Review by mjendza — CHANGES_REQUESTED** [review]

**Comment by mjendza** [comment]
please use https://github.com/airbnb/javascript naming convention

File: script.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+$(document).ready(function () {
+    $.getJSON("http://jsonplaceholder.typicode.com/posts", function (facebookposts) {
+        $.get("templates/facebookposts.hbs", function (template) {
+            let templateScript = Handlebars.compile(template);
```
Comment (goraklaudia) [inline script.js]: Why the name of your template is templateScript?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <script src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-3.3.1.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.11/handlebars.js"></script>
+    </head>
+    <body>
+        <h1 id="faceheading">Facebook</h1>
+        <div class="post">
+            <ul id="postList"></ul>
+        </div>
+        <script type="text/javascript" src="script.js"></script>
```
Comment (mjendza) [inline index.html:14]: why script is inside body ?

File: script.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+            $(facebookposts).each(function (i, value) {
+                $("#postList").append(templateScript(value));
```
Comment (mjendza) [inline script.js]: maybe posts is better name for the HTML container ?

**Review by mjendza — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (otadewosjan) [reply index.html:14]: It's at the bottom of body because I don't want it to load before other html elements.

**Review by goraklaudia — APPROVED** [review]

File: index.html (hunk 5 above)
Comment (mjendza) [reply index.html:14]: Is the body tag (structure) good for scripts ? maybe put it outside the body - at the end of html document ?

Comment (otadewosjan) [reply index.html:14]: According to [mozilla](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/html) only head tag and body tag are allowed inside html tag, and script tag is [permitted in body tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body).

**Review by mjendza — APPROVED** [review]

**Pull request merged**